dScripts/CppScripts.cpp

```diff
@@ -308,6 +314,9 @@ CppScripts::Script* CppScripts::GetScript(Entity* parent, const std::string& scr
 
 	script = invalidToReturn;
 
+	// check for ignored scripts and return early
+	if (std::find(m_IgnoredScripts.begin(), m_IgnoredScripts.end(), scriptName) != m_IgnoredScripts.end()) return script;
```

Comment:
We should not be returning early here.  If returning early then we would not cache these scripts for faster lookup for other objects.  See line 868

Reply:
ah will update.

Reply:
IMHO it's better to just keep the current else if standard we have for script lookups than to iterate over a list, creating unneeded overhead.  At the end of the day it's still just string comparisons so adding a list of strings just seems extra.